<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="EmSystem7" failures="1" tests="5" name="Test" time="564.864" errors="0" timestamp="2021-01-22T21:54:26 EST">
  <testcase classname="com.e7.specFrame.API_FrameWork.testcases.TestCustomerAPI" name="validateCreateCustomerAPIWithInValidAuthKey" time="0.0">
    <skipped/>
  </testcase> <!-- validateCreateCustomerAPIWithInValidAuthKey -->
  <testcase classname="com.e7.specFrame.API_FrameWork.testcases.TestCustomerAPI" name="validateCreateCustomerAPIWithValidData" time="2.336">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at com.e7.specFrame.API_FrameWork.testcases.TestCustomerAPI.validateCreateCustomerAPIWithValidData(TestCustomerAPI.java:115)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- validateCreateCustomerAPIWithValidData -->
  <testcase classname="com.e7.specFrame.API_FrameWork.testcases.TestCustomerAPI" name="validateCreateCustomerAPIWithoutPassingAuthKey" time="0.0">
    <skipped/>
  </testcase> <!-- validateCreateCustomerAPIWithoutPassingAuthKey -->
  <testcase classname="com.e7.specFrame.API_FrameWork.testcases.TestCustomerAPI" name="validateCreateCustomerAPIWithInvalidField" time="0.0">
    <skipped/>
  </testcase> <!-- validateCreateCustomerAPIWithInvalidField -->
  <testcase classname="com.e7.specFrame.API_FrameWork.testcases.TestCustomerAPI" name="validateCreateCustomerAPIWithInValidEmail" time="0.0">
    <skipped/>
  </testcase> <!-- validateCreateCustomerAPIWithInValidEmail -->
</testsuite> <!-- Test -->
